bin/runalyzer-gather

```diff
@@ -185,7 +185,8 @@ def make_name_map(map_str):
     map_re = re.compile(r"^([a-z_A-Z0-9]+)=([a-z_A-Z0-9]+)$")
     for fmap_entry in map_str.split(","):
         match = map_re.match(fmap_entry)
-        assert match is not None
+        assert match and match.group(1) and match.group(2), \
+            "Arguments to -m should have the form F1=FNAME1,F2=FNAME2,..."
```

Comment:
This should be `raise`, not `assert`. (`assert` is only for internal invariants.) (I was young and naive.)